apps/condo/domains/meter/constants/registerMetersReadingsService.js

```diff
@@ -0,0 +1,27 @@
+const READINGS_LIMIT = 500
+
+const DATE_FIELD_PATHS = [
+    { path: 'date', nullable: false },
+    { path: 'meterMeta.verificationDate', nullable: true },
+    { path: 'meterMeta.nextVerificationDate', nullable: true },
+    { path: 'meterMeta.installationDate', nullable: true },
+    { path: 'meterMeta.commissioningDate', nullable: true },
+    { path: 'meterMeta.sealingDate', nullable: true },
+    { path: 'meterMeta.controlReadingsDate', nullable: true },
+]
+
+const DATE_FIELD_PATH_TO_TRANSLATION = {
+    'date': 'meter.import.column.meterReadingSubmissionDate',
```

Comment:
Why can't we just map it like so: 
```
(field) =>  `meter.import.column.${field}.name`
```

Reply:
we would need to map meterMeta.verificationDate to VerificationDate beforehand and so on
we would need to map date to meterReadingSubmissionDate beforehand

So we would just add more mappings